packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/router/ports/ports.factory.js

```diff
@@ -86,10 +87,11 @@ export default /* @ngInject */ (OvhApiXdsl, $translate, TucToast) => {
         );
         return self;
       })
-      .catch(() => {
+      .catch((error) => {
         TucToast.error(
           $translate.instant('xdsl_modem_ports_add_error', {
             name: self.name,
+            reason: error.data && error.data.message ? error.data.message : '',
```

Comment:
Following shorthand might work:

```suggestion
            reason: error.data?.message || '',
```

Could you please confirm me?